frontend/src/components/AddRecipeForm.js

```diff
@@ -11,6 +11,28 @@ const showAlert = () => {
 };
 
 const AddRecipeForm = () => {
+  const [newCategory, setNewCategory] = useState('');
+  const [error, setError] = useState('');
+  async function addCategory(value) {
+    const data = {
+      name: value,
+    };
+
+    console.log(data);
+    data.name !== ''
+      ? setCategoryList([...categoryList, data])
```

Comment:
Itt jobb lenne inkább újra lekérni az API-tól miután sikerült a beküldés, ne a kliens tippelje be, hogy biztos sikerülni fog majd. Mi van, ha elhal a post később?

Reply:
Done.